content/posts/Docker Tags.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Deciding which Docker tag to use"
+date: 2019-03-30T12:00:00Z
+aliases:
+    - /posts/2018-03-30-docker-tags/
```

Comment:
You don't need to add this alias for new posts; I only put it on the old ones when I updated the default URL to not include the date.